modules/xmpp/JingleSessionPC.js

```diff
@@ -1669,7 +1669,8 @@ export default class JingleSessionPC extends JingleSession {
             // NOTE the code below assumes that no more than 1 video track
             // can be added to the peer connection.
             // Transition from no video to video (possibly screen sharing)
-            if (!oldTrack && newTrack && newTrack.isVideoTrack()) {
+            // or transition from camera to share
```

Comment:
please finish the sentence with a period.

Reply:
sure